templates/html_tag/html_tag.component.inc

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * @file
+ * Contains component file.
+ */
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function europa_preprocess_html_tag(&$variables, $hook) {
+  $headers = ['h1', 'h2', 'h3', 'h4', 'h5'];
```

Comment:
Shouldn't we use the old array convention ?

I don't care to use the new one or the old one, it's just that mixing both is crap.
Drupal 7 code should be using old array style, even if we are using PHP >= 5.4.

How can we fix this once for all ?

@gillesdeudon @voidtek @imanoleguskiza @Fefaine